src/functions.php

```diff
@@ -575,6 +575,11 @@ function getQuickdials(array $attributes, bool $alias = false)
  */
 function uploadBackgroundImage($attributes, array $config)
 {
+    if ($config['ftp-disable']) {
```

Comment:
ich würde eigentlich auf die Ausgabe verzichten.

Reply:
Es ist generell keine gute Idee die Logik der Konfigurationsoptionen in jede Funktion reinzuziehen- "enge Koppelung". Also lieber Prüfung vor dem Aufruf?

Reply:
a) ja, aber beim background command (einzige Funktion ist der Upload per ftp) würde bei diasbled ftp der User kein Feedback bekommen, warum der command nix macht nichts macht

b) ich habe schon darauf gewartet, dass Du darüber stolperst :) - war aber vorher schon so. Ich baue es um